.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+**Screenshots**
+
+**Related Issue**
+E.g: *Closes/Fixes #1234*
+
+- [ ]  **Code Quality**
+    - [ ]  The code follows established coding standards and style guidelines
+    - [ ]  The code is properly commented on and provides context when needed (complex feature or hard to understand)
+- [ ]  **Functionality validation**
+    - [ ]  Feature working as described in the related issue
+    - [ ]  Does this PR need environment variables to be set? If yes, have they been set?
+    - [ ]  New API endpoints are properly documented
+    - [ ]  New functions are properly tested (When should we add tests?)
+    - [ ]  Code errors are being logged in the code health service
+- [ ]  **Product validation**
+    - [ ]  UI changes look the same as in Figma
+    - [ ]  Usage events are being sent to the product analytics service
+    - [ ]  New common components have been documented in Storybook
```

Comment:
I worry this might add too much overhead to a feature branch. 

Creating a component and adding it to the storybook, and checking that the colors are defined in figma, etc could be a whole issue/PR in itself. This could also make us more aware of what components we decide to create, outside of what we already have.

This could be part of the issues checklist: If a new component is needed, check it has a ticket specifically for that.

Reply:
It's the same overhead as creating a test for a new service in the backend.

Reply:
my opinion on this is that if you treat design system issues as separate tasks, they never get done. Let's have a reusable mindset when building UIs and try to build custom components as much as possible. In case we have something specific and common that we want to add, let's estimate the effort when estimating the task.